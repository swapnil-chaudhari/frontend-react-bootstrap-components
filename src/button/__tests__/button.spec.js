import React from 'react';
import renderShallow from 'render-shallow';
import Button from '../button';

describe('<Button>', () => {
    describe('when it is rendered with default props', () => {
        let component;

        beforeAll(() => {
            component = renderShallow(<Button />).output;
        });

        test('has proper markup', () => {
            expect(component).toMatchSnapshot();
        });
    });
});
