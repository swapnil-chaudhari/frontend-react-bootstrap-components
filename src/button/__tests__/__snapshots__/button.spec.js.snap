// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Button> when it is rendered with default props has proper markup 1`] = `
<Button
  active={false}
  block={false}
  bsClass="btn"
  bsStyle="default"
  disabled={false}
/>
`;
